kolibri/plugins/coach/assets/src/modules/practiceQuizDetail/actions.js

```diff
@@ -0,0 +1,25 @@
+import { AttemptLogResource } from 'kolibri.resources';
```

Comment:
I think in general, I'd rather not duplicate all the module state for the sake of practiceQuiz - we can do this in the context of the existing Vuex modules (we already have conditional logic in there for exercises and quizzes).